docs/vsphere-golden-image.md

```diff
@@ -144,8 +144,15 @@ To deploy the Windows VM as a reusable image, you have to first generalize the V
 such as installed drivers. Running the `sysprep` command with a *unattend* answer file generalizes the image and 
 makes it ready for future deployments, maintaining all the changes needed for the WMCO installation. 
 
-You can find an example [unattend.xml](unattend.xml) file in this repository. It is just an example and, it **should 
-not** be used directly. You must customize the example to fit your needs.
+A reference [unattend.xml](unattend.xml) file is provided, it **cannot** be used directly. You must customize it
+replacing the `WindowsPassword` parameter with the desired password for the Administrator user. The Password must
+meet complexity requirements as per [Microsoft documentation](https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/password-must-meet-complexity-requirements).
```

Comment:
`password` in lower case please. This is a best practice and not a `must`. So something to the effect of `We suggest following the Microsoft best practices for picking a password...`